pomodoro/src/main/java/pomodoro/service/TeamService.java

```diff
@@ -30,24 +39,46 @@
         return teamsDto;
     }
 
+    @Transactional
+    @SuppressWarnings("unchecked")
+    public List<TeamDto> findUserTeams(Principal principal) {
```

Comment:
Application services should not be aware of the web part of the applicaiton (in this case Principal). The methods in services should accept email or id, instead.